### ISSUE TITLE
Remaining sections are not centered in the New Tab page if other sections are disabled

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [mozilla/activity-stream#3776]
https://bugzilla.mozilla.org/show_bug.cgi?id=1413212

[Affected versions]:
- Firefox 58.0a1 Build ID 20171030103605

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- "browser.search.region" is set to "US" in about:config

[Steps to reproduce]:
1. Open a new tab.
2. Disable the Highlights section and observe the page.

[Expected results]:
- All other section are centered in the page.

[Actual results]:
- The remaining sections remain in place.

[Notes]:
- The issue is also reproducible if you disable the Recommended by Pocket section.
- The issue is not reproducible if you disable both Recommended by Pocket and Highlights
- The issue is not reproducible if you disable Search or Top Sites.
- The issue is not reproducible if you collapse the Recommended by Pocket or Highlights sections.
- Attached a screen recording of the issue: https://goo.gl/Epu2q7
![sections not centered after disable](https://user-images.githubusercontent.com/18304511/32231603-ad54de32-be5e-11e7-9980-2ed615a4ec5a.gif)

**Comment by aaronrbenson** [comment]
@SoftVision-CiprianMuresan does this only happen when the Highlights section is in the empty state? I seem to be getting different behavior when there are highlight items.

**Comment by aaronrbenson** [comment]
We have the layout pin to the top for certain layouts for perceived performance reasons. While it's a little quirky it's working as designed and marking as a wontfix.

**Comment by Mardak** [comment]
This is by design for perceived performance reasons of a prerendered pages. We can revisit with #3367 with dynamically prerendered layout. Marking as blocked to be revisited after #3367.

**Comment by SoftVision-CiprianMuresan** [comment]
@aaronrbenson, sorry for the late response. I can reproduce this behavior with the Highlights section populated as well.
![sections not centered](https://user-images.githubusercontent.com/18304511/32884303-95fbee5e-cac2-11e7-9921-19fbdb8e1872.gif)

**Comment by Mardak** [comment]
Collapsing sections makes the page not prerendered. The page is currently anchored to the top only when it can be prerendered, which is basically when there's no customization of sections (shows search, top sites 1 row, nothing collapsed, highlights OR stories shown)

**Comment by Mardak** [comment]
Moved to https://bugzilla.mozilla.org/show_bug.cgi?id=1413212
